### ISSUE TITLE
db_key::Key` is not implemented for `u64`

### ISSUE THREAD

**Issue by xiuno** [skade/leveldb#46]
I need the u64 as key, but this lib does not support, has the milestone ?
```
let k = 123u64;
let bytes: [u8; 8] = unsafe { std::mem::transmute(i) };
database.put(write_opts, i, &bytes);
```

**Comment by palkeo** [comment]
You can implement a Key type yourself, that implements the Key trait and does what you need, like this:
https://docs.rs/db-key/0.0.5/src/db_key/.cargo/registry/src/github.com-1ecc6299db9ec823/db-key-0.0.5/src/lib.rs.html#1-4

**Comment by xiuno** [comment]
According to the orphan rule, I cannot implement the Key interface in db-key in business code. I have to modify crate db-key directly.

**Comment by Congyuwang** [comment]
Exactly. How to implement this key? I can only copy the entire source code into my library
